src/main/java/adris/altoclef/TaskCatalogue.java

```diff
@@ -218,7 +218,10 @@
             shapedRecipe3x3Block("emerald_block", Items.EMERALD_BLOCK, "emerald");
             shapedRecipe3x3Block("lapis_block", Items.LAPIS_BLOCK, "lapis_lazuli");
             shapedRecipe3x3Block("slime_block", Items.SLIME_BLOCK, "slime_ball");
+            shapedRecipe3x3Block("bone_block", Items.BONE_BLOCK, "bone_meal");
```

Comment:
Upon testing, the bot gets stuck at a crafting table
![image](https://user-images.githubusercontent.com/72313560/148274058-047faef4-5677-4871-aa62-3b477fb22e30.png)
(The lowest item in the task chain flickers.)

Reply:
Additional testing reveals that the bot gets stuck at a crafting table even if it has no bone meal in hand

![image](https://user-images.githubusercontent.com/72313560/148275264-6126cedf-d8ee-4390-b8d6-b4dca5c6000d.png)